Homework2/StackCalculator/StackCalculator/Solution.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace StackCalculator;
+
+public class Solution
+{
+    static void Main()
+    {
+        Console.WriteLine("Please, enter the expression");
+        var inputString = Console.ReadLine();
+        if (inputString == null)
+        {
+            return;
+        }
+        var subs = inputString.Split(' ');
+        Calculator stackCalculator = new Calculator();
+        Console.WriteLine($"{stackCalculator.CountTheExpressionInPostfixForm(subs)}");
+    }
+}
```

Comment:
В C# 10 эти церемонии вообще не нужны:
```suggestion
﻿namespace StackCalculator;

Console.WriteLine("Please, enter the expression");
var inputString = Console.ReadLine();
if (inputString == null)
{
    return;
}
var subs = inputString.Split(' ');
Calculator stackCalculator = new Calculator();
Console.WriteLine($"{stackCalculator.CountTheExpressionInPostfixForm(subs)}");
```